include/core/utilities.h

```diff
@@ -615,4 +617,26 @@ compute_cell_diameter(const double cell_measure, const unsigned int fe_degree)
   return h;
 }
 
+/**
+ * @brief Computes the volume of the cell by integrating after applying
```

Comment:
I see what you mean, but the sentence is a bit difficult to follow. Maybe "Computes the volume of the cell by integrating 1 using its quadrature formula"

Reply:
Yes, I'll write something more similar to the PR description.